### DISCUSSION TITLE
https://plugin-components.vuejs.press/zh/guide/stackblitz.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#529] [Announcements]
# https://plugin-components.vuejs.press/zh/guide/stackblitz.html

在 Markdown 文件中嵌入 StackBlitz 演示。 示例 一个 StackBlitz 项目: 一个自定义设置的 StackBlitz 项目: Props id 类型: string; 必填: 是; StackBlitz id type 类型: 

https://plugin-components.vuejs.press/zh/guide/stackblitz.html

**Comment by ZYMDB** [comment]
## 他喵的，这文档写的怎么这么草率，能不能用点儿心，写全些，写仔细些✍

**Reply by Mister-Hope** [reply]
没人逼着你用。
